python/pyspark/sql/connect/client.py

```diff
@@ -400,6 +400,14 @@ def _execute_and_fetch(self, req: pb2.Request) -> typing.Optional[pandas.DataFra
 
         if len(result_dfs) > 0:
             df = pd.concat(result_dfs)
+            del result_dfs
```

Comment:
why?

Reply:
just want to release the buffer asap, maybe not needed

Reply:
wouldn't be needed.